### PR TITLE
style(conversation): gave the user a bit more context on @

### PR DESCRIPTION
**Pull request by StanGirard**

### CODE REVIEW

File: frontend/public/locales/en/chat.json [hunk 1]
```diff
@@ -12,7 +12,7 @@
     "receivedResponse": "Received response. Starting to handle stream...",
     "errorCallingAPI": "Error calling the API",
     "ask": "Ask a question, or describe a task.",
-    "begin_conversation_placeholder": "Begin conversation here...",
+    "begin_conversation_placeholder": "Begin conversation here, @ to select a brain or / to select a file",
```
Comment (mamadoudicko) [inline frontend/public/locales/en/chat.json:15]: The used one seems to be `actions_bar_placeholder` instead of `begin_conversation_placeholder`

Comment (StanGirard) [reply frontend/public/locales/en/chat.json:15]: It is not

Comment (StanGirard) [reply frontend/public/locales/en/chat.json]: Fixed

File: frontend/public/locales/en/chat.json [hunk 2]
```diff
@@ -12,7 +12,7 @@
     "receivedResponse": "Received response. Starting to handle stream...",
     "errorCallingAPI": "Error calling the API",
     "ask": "Ask a question, or describe a task.",
-    "begin_conversation_placeholder": "Begin conversation here...",
+    "begin_conversation_placeholder": "Begin conversation here, @ to select a brain...",
```
Comment (mamadoudicko) [inline frontend/public/locales/en/chat.json:15]: Is this actually possible now?

It should be introduced by [this PR](https://github.com/StanGirard/quivr/pull/969), which has already updated the placeholder conveniently

**Pull request closed without merging**